### ISSUE TITLE
Customising the HttpClient does not seem to have any effect

### ISSUE THREAD

**Issue by apoorvadixit-ttc** [microspaze/FFImageLoading.Maui#26]
### Description

When I try to add my own HttpClient in the Android MainActivity class, the calls to the source do not seem to carry the custom Http Headers that I added.

### Steps to Reproduce

In MainActivity.cs 
```
internal static HttpClient? httpClient { get; private set; }

protected async override void OnCreate(Bundle? savedInstanceState)
{
    base.OnCreate(savedInstanceState);
    var ffConfig = new FFImageLoading.Config.Configuration
    {
        HttpClient = await GetAuthHttpClient(),
        HttpHeadersTimeout = 300
    };
    var imageService = ServiceHelper.GetService<IImageService>();
    imageService.Initialize(ffConfig);
    //FFImageLoading.ImageService.Instance.Initialize(ffConfig);
    HttpClient client = await GetAuthHttpClient();
}
private async Task<HttpClient> GetAuthHttpClient()
{
    var settingsService = new APIService();
    var deviceToken = await settingsService.GetDeviceToken();
    httpClient = new HttpClient(new AuthenticatedHttpImageClientHandler(settingsService.GetDeviceToken));
    httpClient.DefaultRequestHeaders.Authorization = new System.Net.Http.Headers.AuthenticationHeaderValue(deviceToken);
    return httpClient;
}
```
### Expected Behavior
The calls to the Source as specified in the ffImage should carry the http header but they dont.
Using the Single instance or the ImageService, both dont seem to send the custom HttpHeaders to my API call.
### Basic Information

- Version with issue: 1.1.6

- Platform: Maui,Android

**Comment by microspaze** [comment]
Fixed in the latest version 1.1.8 https://www.nuget.org/packages/FFImageLoading.Maui/1.1.8

Thanks for reporting. Please help to verify the fix when you are free.

**Comment by apoorvadixit-ttc** [comment]
> Fixed in the latest version 1.1.8 https://www.nuget.org/packages/FFImageLoading.Maui/1.1.8
> 
> Thanks for reporting. Please help to verify the fix when you are free.

I can confirm that this is now working.
Thank you so much for fixing it so promptly.
